gramps/gen/db/base.py

```diff
@@ -25,21 +25,21 @@
 from this class.
 """
 
-#-------------------------------------------------------------------------
+# -------------------------------------------------------------------------
```

Comment:
Very strange I'm using the same version of pylint from Ubunutu 14.04(my results below) maybe a difference between windows and linux version? 

Would you please mind removing the space to keep in line with the rest Gramps code so this PR can be accepted?  Thanks

```
$ pylint --version
pylint 1.5.5, 
astroid 1.4.5
Python 3.4.3 (default, Jul 28 2015, 18:20:59) 
[GCC 4.8.4]

$ pylint gramps/gen/db/base.py 
************* Module gramps.gen.db.base
C:1193, 0: Exactly one space required after comma
        name = self.get_table_func(table,"class_func").get_field_alias(name)
                                        ^ (bad-whitespace)
C:1544, 0: Line too long (81/80) (line-too-long)
C:1576, 0: Line too long (82/80) (line-too-long)
C:1617, 0: Exactly one space required after comma
            self.find_backlink_handles(handle,['Person'])]
                                             ^ (bad-whitespace)
C:1641, 0: Wrong continued indentation (add 7 spaces).
                self.find_backlink_handles(family_handle, ['Person'])]
                ^      | (bad-continuation)
C:1682, 0: Wrong continued indentation (add 22 spaces).
                "the family you try to remove is not consistent, please fix "
                ^                     | (bad-continuation)
C:1730, 0: Wrong continued indentation before block (add 4 spaces).
                and ref.role.is_primary()
                ^   | (bad-continuation)
C:1731, 0: Wrong continued indentation before block (add 4 spaces).
                and (birth_ref_index == -1)):
                ^   | (bad-continuation)
C:  1, 0: Too many lines in module (1759/1000) (too-many-lines)
C: 46, 0: Import "from ..lib.childreftype import ChildRefType" should be placed at the top of the module (wrong-import-position)
C: 47, 0: Import "from ..lib.childref import ChildRef" should be placed at the top of the module (wrong-import-position)
C: 48, 0: Import "from .txn import DbTxn" should be placed at the top of the module (wrong-import-position)
C: 49, 0: Import "from .exceptions import DbTransactionCancel" should be placed at the top of the module (wrong-import-position)
W: 76,29: Unused argument 'table' (unused-argument)
W: 76,41: Unused argument 'func' (unused-argument)
R: 76, 4: Method could be a function (no-self-use)
R:1041, 4: Too many arguments (6/5) (too-many-arguments)
R:1131, 4: Too many arguments (10/5) (too-many-arguments)
R: 59, 0: Too many public methods (159/20) (too-many-public-methods)
R:1521, 4: Too many arguments (6/5) (too-many-arguments)
C:1726, 8: Consider using enumerate instead of iterating with range and len (consider-using-enumerate)
E:1748,30: Instance of 'DbWriteBase' has no 'has_changed' member (no-member)
E:1754,16: Instance of 'DbWriteBase' has no 'backup' member (no-member)
E:1755,19: Undefined variable 'DbException' (undefined-variable)
R:1196, 0: Too many public methods (47/20) (too-many-public-methods)
W: 33, 0: Unused import re (unused-import)
W: 34, 0: Unused import time (unused-import)
W: 35, 0: Unused itemgetter imported from operator (unused-import)

<snip>

```